modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java

```diff
@@ -340,6 +340,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			"8.1.0", "8.2.0",
 			new com.liferay.object.internal.upgrade.v8_2_0.
 				ObjectValidationRuleSettingsUpgradeProcess());
+
+		registry.register(
+			"8.2.0", "8.3.0",
+			new com.liferay.object.internal.upgrade.v8_1_0.
+				ObjectRelationshipUpgradeProcess());
```

Comment:
Hey @G-Sa,
Please, verify the path of the upgrade class. Should match the version you described here: `"8.2.0", "8.3.0"`
Then, your class should have been created with the folder: v8_3_0